tests/PageButton.test.tsx

```diff
@@ -4,8 +4,12 @@ import PageButton from '../src/ui/PageButton';
 
 test('render PageButton', () => {
   const onClickHandler = jest.fn();
-  render(<PageButton onClick={onClickHandler}>Page</PageButton>);
-  const pageBtn = screen.getByRole('button', { name: 'Page' });
+  render(
+    <PageButton onClick={onClickHandler} aria-label="Next Page">
+      {'>'}
```

Comment:
Optional: You can mount an icon and assert that it's there, if you want? 

Does this check for "Previous page" as well?

Reply:
Added!